Sleep Tracker/Sleep Tracker/View/SleepButton.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+/// A view that displays a button to control sleep.
+struct SleepButton: View {
+    // MARK: - Public interface
+
+    /// A string value that contains the text on the button. Empty string by default.
```

Comment:
Can we simplify the description? I think it's obvious, that it's a String.
```suggestion
    /// A text on the button. Empty string by default.
```
Please review the rest of doc comments.

Reply:
Fixed.